### ISSUE TITLE
Temp file

### ISSUE THREAD

**Issue by ali-apadana** [tdlib/telegram-bot-api#619]
Hi everyone
When i use getFile(file_id) file is downloading in temp folder. Now How can I find the name of the file being downloaded in the temp folder?

**Comment by levlam** [comment]
You receive file path after downloading is finished. You can't access the file before downloading is completed.

**Comment by ali-apadana** [comment]
Yes. But I need it because I want to create progressbar. I'm coding with php. And I don't have access to system files such as cpp files

**Comment by levlam** [comment]
This is not possible to do.
